src/index.js

```diff
@@ -58,26 +58,34 @@ class ActiveStorageProvider extends React.Component<Props> {
     }
   }
 
+  _getHeaders = (): CustomHeaders => {
+    // fastest performance for indifferent access: https://jsperf.com/downcase-and-merge-object-keys
+    const mappedHeaders = {}, { headers = {} } = this.props
```

Comment:
Agreed that the default there is unnecessary, but I think an explicit coalescing here is better as passing a falsey value to the headers prop (whether explicitly or through wrapper components) shouldn't break things in my opinion (i.e. `<ActiveStorageProvider headers={false} />`)

Reply:
This surprises me. Why wouldn't it break things to pass a value of the wrong type?

Reply:
thinking about it more, I think that would be a case where we want an error to be raised

Reply:
Yeah okay I see where you were coming from. You’re not crazy haha. But since `headers={false}` would read as opting out of any headers at all, which doesn’t really make sense IMO, I think the “defaultProps/undefined behavior if we’re passed the wrong type” plan is the way to go 👍